goinstant/godogs_test.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/cucumber/godog"
+	"github.com/pkg/errors"
+)
+
+var (
+	binaryFilePath string
+	logs           string
+	directoryNames []string
+)
+
+func theCommandIsRun(command string) error {
+	res, err := runTestCommand(binaryFilePath, strings.Split(command, " ")...)
```

Comment:
Hmm, this seems prone to failure, I know it's highly unlikely, but a subpart from a command should be able to have a space in it, so is it possible to change the command parameter to be a slice of type string instead of needing to do the `strings.Split` here?

Reply:
I'll speak under correction, but I don't think it is possible. I think it's just how gherkin passes the info through...

Reply:
Okay, guess we'll have to live with it and figure out another solution if it ever becomes a problem.